### PR TITLE
Feature/dockerize

### PR DESCRIPTION
**Pull request by aschiavon91**
Just adding a simple docker configuration.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM python:3.7
```
Comment (mateusmaaia) [inline Dockerfile]: There isn't a scratch image? something to make it smaller, you can also do it in a multistage if you need this image to build

Comment (aschiavon91) [reply Dockerfile]: good point and thanks for this idea.
What do u think about an alpine image?
The size of `3.7-alpine` tag is something around 35mbs.

Comment (mateusmaaia) [reply Dockerfile]: Seems good!

File: README.md [hunk 2]
```diff
@@ -79,6 +79,15 @@ Each test can be executed through [unittest](https://docs.python.org/3/library/u
 make test
 ```
 
+<a name="docker"></a>**Docker**
+
+To successful run docker container, the `DOMAINS` build-arg must be passed in docker build command.
```
Comment (mateusmaaia) [inline README.md]: If you are thinking about running it locally on docker, that will be a good idea to use `docker-compose`, or even easier: just add this commands to the Makefile

Comment (aschiavon91) [reply README.md]: another great idea, thanks man!

File: README.md [hunk 3]
```diff
@@ -79,6 +79,15 @@ Each test can be executed through [unittest](https://docs.python.org/3/library/u
...
+Ex:
+```
+docker build --build-arg DOMAINS="grupozap.com" -t n0s3p4ss .
```
Comment (julianoborba) [inline README.md]: Add sample for multiple domains also, guess it will be usefull.

Comment (aschiavon91) [reply README.md]: seems very helpful, i will do this update, thanks!

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ARG DOMAINS
+ENV TARGET_DOMAINS=$DOMAINS
+
+RUN mkdir /app
+WORKDIR /app
+COPY . .
+RUN apk add build-base g++ git libffi-dev make nmap openssl-dev tor
+RUN make install
+
+CMD echo $TARGET_DOMAINS | sed "s/\(.*\)/'\1'/;s/,/' '/" | xargs -t pipenv run python3 main.py --domains
```
Comment (aschiavon91) [inline Dockerfile]: If anyone has a better way to do this, pls tell me

**Pull request merged**